app/lib/services/voip/MediaSessionInstance.ts

```
@@ -89,9 +101,40 @@ class MediaSessionInstance {
+							voipDebugLog('sendSignal', 'login wait timed out, dropping', { type: (signal as any)?.type });
+							return;
+						}
+						voipDebugLog('sendSignal', 'login ready, proceeding');
+					}
+					await sdk.methodCall('stream-notify-user', `${userId}/media-calls`, JSON.stringify(signal));
+					voipDebugLog('sendSignal', 'methodCall resolved', { type: (signal as any)?.type });
+					logVoipFirstSignalElapsed();
+				} catch (e: unknown) {
+					voipDebugLog('sendSignal', 'methodCall rejected', String(e));
+				}
+			})();
```

ESLint

[error] Expected 'undefined' and instead saw 'void'.

(no-void)